src/api/TKRecoveryManager.h

```diff
@@ -0,0 +1,62 @@
+//
+//  Created by Sibin Lu on 10/20/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TKTypedef.h"
+
+@class TKUnauthenticatedClient;
+@protocol TKCryptoEngineFactory;
+@class GatewayService;
+@class TKRpcErrorHandler;
+
+/**
+ * Use this class to manage recovery process.
```

Comment:
Is it the account recovery process? Unclaimed funds recovery process? Reading along, I think "account". Would be good if comment said so. (...and maybe if class name said so but maybe we have plans to also use this class to recover other things in the future?)

Reply:
Good point. Update the methods and comments to "Member Recovery"